src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -103,6 +103,19 @@ internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord)
                 otlpLogRecord.Flags = (uint)logRecord.TraceFlags;
             }
 
+            if (logRecord.StateValues != null)
+            {
+                foreach (var stateValue in logRecord.StateValues)
+                {
+                    var attributeKey = stateValue.Key;
+                    var attributeValue = stateValue.Value?.ToString(); // TODO: Is ToString() correct here?
```

Comment:
consider this as a ref:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs#L205-L237

I don't think the ToString() should be ever used.